src/App.js

```diff
@@ -60,13 +84,20 @@ export function App () {
         setAppMode(APP_STATE.FILE_VIEW);
     };
 
+    useEffect(() => {
+        console.debug("Version:", config.version);
+        const lsTheme = localStorage.getItem("ui-theme");
+        switchTheme(lsTheme === THEME_STATES.LIGHT?THEME_STATES.LIGHT:THEME_STATES.DARK);
```

Comment:
* Space before and after `?` and `:`
* Prefer constant === variable

Reply:
Done